distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1356,3 +1286,8 @@ print_table header rows indices_count format_term =
 Table.from (that : Text) (format:File_Format.Delimited|File_Format.Fixed_Width = File_Format.Delimited '\t') (on_problems:Problem_Behavior=Report_Warning) =
     java_reader = StringReader.new that
     Delimited_Reader.read_from_reader format java_reader on_problems
+
+Text.from (that : Table) (format:File_Format.Delimited|File_Format.Fixed_Width = File_Format.Delimited '\t') =
+    java_writer = StringWriter.new
```

Comment:
I wonder if we should encapsulate this in the Delimited_Writer

Reply:
Then the `from` function will just only be an alias for that.

But I think that's indeed a good approach - much like `table.write` does. I'll move this, and same for `Table.from` then, to be consistent.